Documentation/Contributing/development-flow.md

```diff
@@ -196,6 +196,58 @@ Rebasing is a very powerful feature of Git. You need to understand how it works
 * If there are merge conflicts, you will be prompted to fix them before continuing. Read the output closely. It will tell you how to complete the rebase.
 * When done rebasing, you will see all of your commits in the history.
 
+### Testing changes and debugging issues in Ceph manager modules
+
+The Ceph manager modules are written in Python and can be individually and dynamically loaded from the manager. We can take advantage of this feature in order to make easier to test changes and to debug issues linked with this kind of module.
+This is just a "trick way" to test/debug quickly and easily any modification in manager modules, but do not replace the right way to do the modification, which is the build of a new Ceph container including your modifications.
+
+The ceph dashboard and the rook orchestrator modules are probably the two modules more prone to have modifications affecting the rook cluster, and therefore it is interesting to know how to debug and test changes easily in these modules.
+
+The way to proceed is to make the modification directly in the manager module and reload it:
+
+1. Find out what is your active manager pod and shell into it.
```

Comment:
This will simplify some of the instructions if we just say to use a single mgr instead of worrying about the active mgr and failing over.
```suggestion
1. Update the cluster so only a single mgr pod is running. Set the `mgr.count: 1` in the CephCluster CR if it is not already.
```